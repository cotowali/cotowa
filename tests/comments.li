// comment
assert 1 == 1 // comment

/* multi
 * line
 * comment
 */

assert /* comment */ 1 /* comment */ == /* comment */ 1 /* comment */

/* comment
// line comment in block comment */ assert 1 == 1
