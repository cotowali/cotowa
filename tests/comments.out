this is not a block comment
